src/store.js

```diff
@@ -450,4 +475,18 @@ const onHelperResult = function(response) {
     this.highlightPreTag,
     this.highlightPostTag
   );
+
+  if (!this._helper.hasPendingRequests()) {
+    clearTimeout(this._stalledSearchTimer);
+    this._stalledSearchTimer = null;
+    this.isSearchStalled = false;
+  }
+};
+
+const onHelperSearch = function() {
+  if (!this._stalledSearchTimer) {
```

Comment:
not sure if this is a big deal, but maybe it could not be done only if the stalled search feature is enabled? Might be hard to find out about it here...

Reply:
The feature is always there, for simplicity sake.